peerconnection.go

```diff
@@ -2092,13 +2096,41 @@ func (pc *PeerConnection) writeRTCP(pkts []rtcp.Packet, _ interceptor.Attributes
 	return pc.dtlsTransport.WriteRTCP(pkts)
 }
 
-// Close ends the PeerConnection
+type shouldGracefullyClose bool
+
+const (
+	shouldGracefullyCloseNo  shouldGracefullyClose = false
+	shouldGracefullyCloseYes shouldGracefullyClose = false
```

Comment:
should this be true?

Reply:
It's a matter of style, but this pattern is easier/more popular IMO

```
return pc.close(false /* shouldGracefullyClose */)
```

Reply:
yep and sure good idea